asm/macros/battle_script.inc

```diff
@@ -1518,6 +1518,21 @@
 	.4byte \ptr
 	.endm
 
+	.macro trytrainerslidezmovemsg, battler:req
+	callnative BS_TryTrainerSlideZMoveMsg
+	.byte \battler
```

Comment:
The battler argument isn't needed. You can just use `gBattlerAttacker` inside the callnative. E.g `BS_SetGlaiveRush`